src/components/snap-camera-button.js

```diff
@@ -0,0 +1,8 @@
+AFRAME.registerComponent("snap-camera-button", {
```

Comment:
Since these are so tightly coupled anyway I would just have `camera-tool` handle this instead and ditch this component. This is how `media-video` handles its overlay.